overlay/mkcrate.nix

```diff
@@ -236,7 +243,11 @@ let
     installPhase = ''
       mkdir -p $out/lib
       cargo_links="$(remarshal -if toml -of json Cargo.original.toml | jq -r '.package.links | select(. != null)')"
-      install_crate ${host-triple} ${if release then "release" else "debug"}
+      if (( MINOR_RUSTC_VERSION < 41 )); then
+        install_crate ${host-triple} ${if release then "release" else "debug"}
+      else
+        install_crate2 ${host-triple}
```

Comment:
`install_crate2` doesn't need the `${host-triple}` argument so we can safely remove it.

Reply:
Sure! I'll do that in a follow-up PR.